### ISSUE TITLE
Adding cryoCare to Scipion?

### ISSUE THREAD

**Issue by pconesa** [juglab/cryoCARE_T2T#5]
Dear developers, we are working on a tomography set of plugins for tomography.

We have imod, Eman, dynamo novaCTF, motincor2 ctffind4, gCtf...deepfinder just joined the project.

There isn't a denoiser, you can be the first one. Is this something you will be interested in.

You'll count with our assistance, of course.

**Comment by tibuch** [comment]
Hi @pconesa 

Thank you for reaching out. That sounds great! We would love to contribute and see cryoCARE as part of Scipion.

Could you recommend a Scipion entry point for developers?

Best wishes

**Comment by pconesa** [comment]
Great! @tibuch ...

I'd say for https://scipion-em.github.io/docs/docs/developer/creating-a-plugin would be ok for Scipion 2 plugins.

We need to work on scipion3 for that you'll need to install scipion3:

Try this--> https://pypi.org/project/scipion-installer/

And then create a plugin based on the plugin template (branch python3_migration):
https://github.com/scipion-em/scipion-em-template/blob/python3_migration/README.rst
 
Please, bear in mind that we are almost betatesting new release so there is missing "stability" and documentation. Nevertheless, this should not stop us. We are adding to our slack workspace all developers to get immediate response.

You and the rest are very welcome to join us. It would be the best way to kickoff this, maybe also arranging a TC.

**Comment by tibuch** [comment]
@fjug and myself would like to join your slack workspace and a kickoff meeting sounds great.

Looking forward!

**Comment by tibuch** [comment]
I just watched the webinar about Scipion at CCP-EM Online Symposium and it was very informative. Based on what I saw, I think it makes sense to focus on getting cryoCARE to run with Scipion 3.0. The fact that the tomographic possibilities are still under development sounds also interesting and I would be happy to try out to convert our current cryoCARE pipeline to Scipion 3.0 and test it out. Having cryoCARE as a Scipion 3.0 pipeline sounds like a much more stable and reliable deployment strategy than the one we are following now. 

How could we join the slack channel? (I am a novice slack user, most of my communication so far is over gitter.)

**Comment by pconesa** [comment]
Great! I'm glad Jose Miguel's talk had that positive influence!

This is an invitation link to slack: it will last 7 days and you need to remove the last "X" (avoiding bots)

https://join.slack.com/t/scipion/shared_invite/zt-eayhkcsw-81sdDgkkiHDhL4MC2ol2~QX

I think this is the right timing, other developers are joining...see you at slack!

**Comment by pconesa** [comment]
Hi, yesterday one of our tomography betatesters asked about cryoCARE. 
Are you still willing to integrate it into Scipion.

Current status of scipion3: beta almost ready. 
Tomography: All is becoming smooth and our developers and betatesters have already managed to run decent workflows combining: imod, eman, dynamo, relion, novactf....

If you want we can have a video call to discuss details?

**Comment by tibuch** [comment]
Hi @pconesa,

Yes, we still want to integrate cryoCARE into Scipion.

A video call would be great to get some pointers on how to integrate cryoCARE the most efficient way into Scipion. I am currently in my last year of my PhD and things start to accumulate :slightly_smiling_face:

**Comment by pconesa** [comment]
Great, I'd recommend the following:

Install scipion3: https://scipion-em.github.io/docs/scipion3_instalation/docs/scipion-modes/install-from-sources.html

Then as developer resources you have:
https://scipion-em.github.io/docs/scipion3_instalation/docs/developer/developers.html

And an starting point: https://scipion-em.github.io/docs/scipion3_instalation/docs/developer/creating-a-plugin

In any case, after installing scipion3 we could arrange a video call to define next steps.

What we did with the developer of deepfinder worked very good. This was, having a call to understand deepfinder and choose the simplest method. We then created a seed if the plugin with that method implemented and from there he integrated the rest having some calls and constant availability through slack.
